tasks/release.js

```diff
@@ -7,38 +7,40 @@ const packager = require('electron-packager')
 const fs = require('fs-extra')
 var glob = require('glob')
 var JSZip = require('jszip')
+const minimist = require('minimist')
 const zlib = require('zlib')
 var tar = require('tar-stream')
 var duplexer = require('duplexer')
 const packageJson = require('../package.json')
 
-let skipPack = false
-let binaryPath = null
-process.argv.forEach(function (val) {
-  if (val === '--skip-pack') {
-    console.log('Skipping packaging')
-    skipPack = true
-  }
-  if (val.startsWith('--binary')) {
-    binaryPath = val.replace('--binary=', '')
-    fs.accessSync(binaryPath)
-    console.log('Using prebuilt binary', binaryPath)
-  }
-})
-
-if (!binaryPath) {
+const {
+  binary: binaryPath,
+  platform,
+  'skip-pack': skipPack
+} = minimist(
+  process.argv.slice(2),
+  { default: { platform: `darwin,linux,win32` } }
+)
+
+if (binaryPath) {
+  fs.accessSync(binaryPath)
+  console.log('Using prebuilt binary', binaryPath)
+} else {
   console.error(`\x1b[31mPlease specify a gaia binary for this platform using the "--binary" flag
-    Example: npm run build:darwin -- --binary=./gaia
+    Example: node tasks/release.js --binary=./gaia
```

Comment:
shouldn't this be `yarn run build --platform=darwin --binary=./gaia`?

Reply:
No, `yarn run build` and `node tasks/release.js` are two different processes with two different sets of arguments.  `yarn run build` creates a Docker container and runs `node tasks/release.js` within it.

Reply:
Ok cool. But wouldn't the dev be confused, if he did a mistake in `yarn run build` and gets a recommendation about `node tasks/release.js`? How about catching this error in the build script then?